src/tests/test_scripts/btree_test.py

```diff
@@ -38,33 +40,44 @@
         log_mods = arg
         print("log_mods (%s)" % arg)
     if opt in ('-f', '--fibers'):
-        fibers = " --n_fibers=" + arg
+        fibers = " --num_fibers=" + arg
         print("number of fibers per thread (%s)" % arg)
     if opt in ('-p', '--preload_size'):
         preload_size = " --preload_size=" + arg
         print("preload_size = (%s)" % arg)
     if opt in ('-t', '--threads'):
-        threads = " --n_threads=" + arg
+        threads = " --num_threads=" + arg
         print("number of threads (%s)" % arg)
     if opt in ('-n', '--num_entries'):
         num_entries = " --num_entries=" + arg
         print("number of entries (%s)" % arg)
     if opt in ('-i', '--num_iters'):
         num_iters = " --num_iters=" + arg
         print("number of iterations (%s)" % arg)
+    if opt in ('-r', '--run_time'):
+        run_time = " --run_time=" + arg
+        print("total run time (%s)" % arg)
+    if opt in ('-v', '--dev_list'):
+        dev_list = arg
+        print(("device list (%s)") % (arg))
 
 operations = ""
 if bool(op_list and op_list.strip()):
     operations = ''.join([f' --operation_list={op}' for op in op_list.split()])
 
-btree_options = num_entries + num_iters + preload_size + fibers + threads + operations
+addln_opts = ' '
+if bool(dev_list and dev_list.strip()):
+    addln_opts += ' --device_list '
+    addln_opts += dev_list
+
+btree_options = num_entries + num_iters + preload_size + fibers + threads + operations + run_time + addln_opts
 
 
 def normal():
```

Comment:
Do you plan to add the whatever hari suggested in this PR or separate PR ? --level [short|medium|large]

Reply:
This is not regarding this pr . later